controller/pagecontroller.php

```diff
@@ -96,6 +92,35 @@ public function index() {
 	}
 
 	/**
+	 * Returns the parameters to be used in the index function
+	 *
+	 * @param $appName
+	 *
+	 * @return array<string, string>
+	 */
+	private function getParameters($appName) {
```

Comment:
Also, this is a private method. The order in PHP classes for this project is

public
protected
private

So this method should be moved down as the first private method, before `showPublicPage`